sale_pricelist_auto_update/models/res_partner.py

```diff
@@ -59,11 +59,16 @@ def _update_current_pricelist(self):
                     _("Cannot find a pricelist that matches the conditions."))
         return True
 
-    def _get_customer_ids(self):
-        partners = self.env['res.partner'].sudo().search(
-            [('customer', '=', True),
-             ('active', '=', True)]
-        )
+    def _get_customer_ids(self, partner_id=False):
```

Comment:
Would be good to mention the problematic case in case `commercial_partiner_id` is not used - if a company has more than one contact and each contact has its own sales order(s), we need to aggregate the sales amounts of all the contacts  to get the sales amount for the company.